src/renderers/webgl/WebGLInfo.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+function WebGLInfo( gl ) {
+
+	var memory = {
+		geometries: 0,
+		textures: 0
+	};
+
+	var render = {
+		frame: 0,
+		calls: 0,
+		faces: 0,
+		points: 0,
+		segments: 0
+	};
+
+	function update( count, mode, instanceCount ) {
+
+		instanceCount = instanceCount || 1;
+
+		render.calls ++;
+
+		switch ( mode ) {
+
+			case gl.TRIANGLES:
+				render.faces += instanceCount * ( count / 3 );
+				break;
+
+			case gl.LINES:
+				render.segments += instanceCount * ( count / 2 );
+				break;
+
+			case gl.LINE_STRIP:
+				render.segments += instanceCount * ( count - 1 );
+				break;
+
+			case gl.LINE_LOOP:
+				render.segments += instanceCount * count;
+				break;
+
+			case gl.POINTS:
+				render.points += instanceCount * count;
+				break;
+
+			default:
+				console.error( 'THREE.WebGLInfo: Unknown draw mode:', mode );
```

Comment:
If I'm right, this console error message shows up each render call if user sets a wrong mode. That means the tons of the same error message will be displayed on user's console until user corrects the mode.

Any solutions to avoid that?
Just ignoring wrong mode would be one of the options?

Reply:
Well, we've ignored it before so we can of course remove the `default` block. On the other hand, rendering with a invalid primitive is something that should actually never happen. I think it wouldn't be too bad if we produce error messages in this case.

Reply:
Ah, right. I've read through `WebGLRenderer` and it seems that a wrong mode won't be set even user sets a wrong `drawMode` to `Mesh`. So I think it's ok to leave this console.error now.